app/pub_sub/subscribers/base.rb

```diff
@@ -0,0 +1,25 @@
+module Subscribers
+  class Base
+    class_attribute :subscriptions_enabled
```

Comment:
My fault, the idea is to disable subscriptors during the tests, unless we want to test them explicitly. 

This is probably premature optimization, but I remember how slow was Movida test suite because of the Activity tracking.